.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dockerFile": "dockerfile",
+    "context": "..",
```

Comment:
Just wondering if we need to set the "context"? I think we don't do it in our other repos.

Reply:
The default context is the directory of the `dockerfile` (in this case, it is thus `.devcontainer/`).

But I need to mount the `requirements` directory.

So yes, it's needed in this case.

Reply:
Ok! Got it!